src/components/BasicTabs/index.tsx

```
@@ -42,37 +49,95 @@ export default function BasicTabs() {
+              key={vaga.id}
+              id={vaga.id}
+              image={vaga.image}
+              enterprise={vaga.enterprise}
+              position={vaga.position}
+              applicationDate={vaga.applicationDate}
+              closingDate={vaga.closingDate}
+              status={vaga.status}
+              favorite={vaga.favorite}
+              onFavoriteToggle={fetchData}
+            />
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)